### PR TITLE
Bugfix - Typos

### PR DESCRIPTION
**Pull request by jutaz**
This PR fixes few typo found around the place

### CODE REVIEW

**Comment by jutaz** [comment]
I signed it!

File: .travis.yml [hunk 1]
```diff
@@ -47,7 +47,7 @@ install:
   # Log HTTP requests
   - npm config set loglevel http
   - npm install -g npm@2.5
-  # Instal npm dependecies and ensure that npm cache is not stale
+  # Instal npm dependencies and ensure that npm cache is not stale
```
Comment (Narretz) [inline .travis.yml]: There' also *instal* -> *install*

Comment (jutaz) [inline .travis.yml]: Haha, missed this one!

**Comment by jutaz** [comment]
Amended to include `Instal` -> `Install`

File: src/ngAnimate/animateCss.js [hunk 2]
```diff
@@ -261,7 +261,7 @@ function computeCssStyles($window, element, properties) {
       }
 
       // by setting this to null in the event that the delay is not set or is set directly as 0
-      // then we can still allow for zegative values to be used later on and not mistake this
+      // then we can still allow for negative values to be used later on and not mistake this
```
Comment (gkalpak) [inline src/ngAnimate/animateCss.js]: Maybe `zegative` means `zero or negative` :stuck_out_tongue:

Comment (jutaz) [inline src/ngAnimate/animateCss.js]: But from what I understand, zero values are being set to `null`, right? So in that case zero is not allowed :wink:

Comment (gkalpak) [inline src/ngAnimate/animateCss.js]: Yes, but it could be bug in code and not a typo in docs :stuck_out_tongue:
Just kidding...

**Comment by gkalpak** [comment]
Nice one !

I favour (pun intended) `cancelled` / `behaviour` over `canceled` / `behavior`, but that's unimportant :)

File: src/ngAnimate/animateCss.js (hunk 2 above)
Comment (jutaz) [inline src/ngAnimate/animateCss.js]: Haha, that would be bad

**Comment by jutaz** [comment]
> I favour (pun intended) cancelled / behaviour over canceled / behavior, but that's unimportant :)

Yeah, but given that this project is `EN_us`, it makes sense, as `behaviour` and `cancelled` are more from `EN_gb` locale :wink:

**Comment by gkalpak** [comment]
BTW this is not "backportable" to `v1.4.x`.
If you care to resolve the conflicts and submit a PR against the `v1.4.x` branch (since you seem to be really serious about typos), I'd be happy to merge :smiley:

Thx for all the typo crashing so far :+1:

**Pull request closed without merging**